libs/git/repository.go

```diff
@@ -160,6 +160,11 @@ func (r *Repository) Ignore(relPath string) (bool, error) {
 		trailingSlash = "/"
 	}
 
+	// Never ignore the root path (an unnamed path)
```

Comment:
Not all directories will end with a `/` anyway. With `foo/bar` in an ignore file, it can be a file or a directory and will be ignored just the same. The nuance is that the ignore pattern may end with a `/` to indicate it's a directory. That's why the trailing slash needs to be kept around. This code doesn't actually do filesystem calls to check if something is a file or a directory; it's all string manipulation and comparison.